doc/source/user_guide/text.rst

```diff
@@ -618,3 +618,4 @@ Method Summary
     :meth:`~Series.str.istitle`;Equivalent to ``str.istitle``
     :meth:`~Series.str.isnumeric`;Equivalent to ``str.isnumeric``
     :meth:`~Series.str.isdecimal`;Equivalent to ``str.isdecimal``
+    :meth:`~Series.str.casefold`;Equivalent to ``str.casefold``
```

Comment:
not sure if these are ordered here - if they r alphabetic then this is not in the right place; otherwise maybe put next to lower

Reply:
ok, the methods in `series.rst` is alphabetically ordered, here is not. But I will move it up a bit!